test/validators.mbtiles.test.js

```diff
@@ -111,6 +111,7 @@ test('lib.validators.mbtiles: file too big', function(t) {
     t.ok(err, 'expected error');
     t.equal(err.code, 'EINVALID', 'expected error code');
     t.equal(err.message, expected.mbtilesErrors.filetoobig, 'expected error message');
+    console.log('made it here?');
```

Comment:
Yes, I think this happens. There are several places where the callback is fired and there are likely to be bugs where callbacks are fired too soon or more than once.